modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -50,11 +50,10 @@ For more information, see link:https://www.apollographql.com/docs/react/essentia
 === Global Update Functions
 
 Apollo client holds all mutation parameters in memory.
-Offline Client will save and restore all mutations with exact parameters
-when application is restarted. Update functions supplied to mutations cannot be saved in the cache. 
-As result all optimisticResponses will disappear from application after restart.
+An offline Apollo client will continue to store mutation parameters and once restarted, it will restore all mutations to memory. Any Update Functions that are supplied to mutations cannot be cached by an Apollo client resulting in the loss of all optimisticResponses after a restart. _Update functions_ supplied to mutations cannot be saved in the cache. 
```

Comment:
This text suggests that an offline Apollo client is stopped, do you mean 'and once online'?

are functions supplied to mutations? just wondering about the choice of verb, maybe 'applied'?

Reply:
I understand your point but don't agree about it suggesting it's stopped. Google's definition of offline is "not controlled by or directly connected to a computer or the Internet". I do agree that 'and once online' is better than the word 'restarted'.

I don't have the technical insight to respond to the second question.

Reply:
See below.